src/components/Inputs/StandardInput/StandardInput.tsx

```diff
@@ -1,28 +1,40 @@
 import { Box } from "@mui/system";
+import { FieldErrors } from "react-hook-form";
+
+import { ErrorValidationText } from "../../../validation/styled";
 
 import { StyledTextField, StyledLabel } from "./styled";
 
 interface IStanderdInputProps {
   text: string;
   registerName: string;
   label: string;
-  register: (registerName: string) => {};
+  options: Object;
+  register: (registerName: string, options: Object) => {};
+  errors: FieldErrors;
 }
 
 const StandardInput = ({
   text,
   register,
   label,
   registerName,
+  options,
+  errors,
 }: IStanderdInputProps) => {
+  const errorMessage = errors[registerName]?.message;
```

Comment:
You don't need yo pass all errors, just pass exact error for this field, other are unnecessary

Reply:
done